lib/common_widgets/list_cocktail.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+
+class ListCocktail extends StatelessWidget {
+  final List<dynamic> list;
+
+  ListCocktail({required this.list});
+
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+      child: ListView.builder(
+          scrollDirection: Axis.vertical,
+          shrinkWrap: true,
+          itemCount: list.length,
+          itemBuilder: (_, index) {
+            final String name = list[index].name;
+            return Container(
+              padding: const EdgeInsets.all(10.0),
+              margin: const EdgeInsets.all(10.0),
```

Comment:
Can we add these constants inside the `Spacing` file as it appears in the `ReadMe` so that it can be standardized and reused in a class of constants dedicated to this classic case of use.
```suggestion
              padding: const EdgeInsets.all(Spacing.sm),
              margin: const EdgeInsets.all(Spacing.sm),
```
You can use the variables name that makes more sense for you in this project.